utils/install_precompiled_kernels.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+ROCMINFO=$(which rocminfo)
+if [ -z "$ROCMINFO" ];
+then
+    ROCMINFO=/opt/rocm/bin/rocminfo;
+fi
+
+if [ -f $ROCMINFO ];
+then
+    echo "using rocminfo at $ROCMINFO"
+else
+    echo "rocminfo is not installed, please install the rocminfo package"
+    return -1
+fi
+arches=$($ROCMINFO | grep -e ' gfx' -e 'Compute Unit:' | awk '/Name/{ arch= $2} /Compute Unit:/ {if(arch != "") { all_arches[(arch "-" $3)] }} END { for (a in all_arches) { print a}  }')
+
+while IFS= read -r line ; 
+do 
+    if [ -f /etc/redhat-release ]; then
```

Comment:
Instead of checking for redhat, could you check for the `yum` command?

Reply:
I can install yum on ubuntu, but that does not make it Ubuntu  ?